src/main/java/fi/aalto/cs/apluscourses/intellij/actions/CourseProjectAction.java

```diff
@@ -161,6 +165,25 @@ private Course tryGetCourse(@NotNull Project project, @NotNull URL courseUrl) {
     }
   }
 
+  private void startAutoInstalls(@NotNull Project project,
```

Comment:
writing (unit) tests has a hidden agenda - testability enforces simplicity. Therefore, I would really love to see test cases covering this method's logics as a whole or piece-by-piece (it's totally fine to make it `public` to be testable)

Reply:
I should've seen this coming :smile: But you're absolutely right, I'll add tests :+1: